tests/test_interpolation.py

```diff
@@ -499,6 +501,20 @@ def foo() -> int:
         OmegaConf.register_new_resolver("foo", lambda: 10)
 
 
+try:
+    inspect.signature(print)
+    print_is_inspectable = False
+except ValueError:
+    print_is_inspectable = False
```

Comment:
If we find an example of a function that is always un-inspectable, let's use it.
otherwise, good to go.

Reply:
I looked a bit (at the doc of `inspect.signature()`, at a way to create a non-inspectable object in Python, and at the content of a function to see if I could delete some attributes to fool `inspect`), but couldn't easily find something. So that's the best I could come up with.

Reply:
This looks like a good use case for mocking the function to raise the expected exception.

Reply:
Good idea, let me give it a shot

Reply:
Done in 8c67b8c